test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/RejectCaseControllerSpec.scala

```diff
@@ -227,7 +227,7 @@ class RejectCaseControllerSpec extends WordSpec with Matchers with UnitSpec
     }
 
     "redirect to a default page if the status is not right" in {
-      when(casesService.suspendCase(refEq(caseWithStatusOPEN), refEq(operator))(any[HeaderCarrier])).thenReturn(successful(caseWithStatusREJECTED))
+      when(casesService.rejectCase(refEq(caseWithStatusOPEN), refEq(operator))(any[HeaderCarrier])).thenReturn(successful(caseWithStatusREJECTED))
```

Comment:
Will need (, any[FileUpload], any[String]) passing into rejectCase call, but guess you'll have already spotted as it won't compile